Description: Switch to using gtksourceview-4
 gtksourceview-3 is no longer supported.  This patch ensures
 desktopfolder compiles with the supported gtk3 v4 variant.
Author: David Mohammed <fossfreedom@ubuntu.com>
Bug-Debian: https://bugs.debian.org/1079409
Forwarded: not-needed
Last-Update: 2024-08-24

--- desktopfolder-1.1.3.orig/meson.build
+++ desktopfolder-1.1.3/meson.build
@@ -92,7 +92,7 @@ executable(
         dependency('json-glib-1.0'),
         dependency('gdk-pixbuf-2.0'),
         dependency('libwnck-3.0'),
-        dependency('gtksourceview-3.0'),
+        dependency('gtksourceview-4'),
         m_dep
     ],
     install : true
